cmd/entrypoint/waiter.go

```diff
@@ -31,11 +32,14 @@ func (rw *realWaiter) setWaitPollingInterval(pollingInterval time.Duration) *rea
 //
 // If a file of the same name with a ".err" extension exists then this Wait
 // will end with a skipError.
-func (rw *realWaiter) Wait(file string, expectContent bool, breakpointOnFailure bool) error {
+func (rw *realWaiter) Wait(ctx context.Context, file string, expectContent bool, breakpointOnFailure bool) error {
 	if file == "" {
 		return nil
 	}
 	for ; ; time.Sleep(rw.waitPollingInterval) {
+		if ctx.Err() != nil {
+			return nil
```

Comment:
```suggestion
			return err
```

Or probably ignore it `if !errors.Is(err, context.Canceled)`

Reply:
Thank you for the review. Fixed.